base/complex.jl

```diff
@@ -524,6 +630,12 @@ function log1p{T}(z::Complex{T})
     end
 end
 
+"""
+    ^(x, y)
+
+Exponentiation operator.
+"""
+Base.:(^)(x, y)
```

Comment:
Adding that  line was intentional. Without the line `Base.:(^)(x, y)`, moving the docstring for exponentiation inline causes Documenter's output to include the message `!! No docs found for 'Base.:^(::Any, ::Any)'. [src/stdlib/math.md]`. 

Do you know if this message suggests a real problem? You can get the docstring for exponentiation at the REPL in any case.

Thanks!